src/main/scala/Column.scala

```diff
@@ -245,12 +278,47 @@ object Column {
       titles = columnProperties.get("titles"),
       suppressOutput = getSuppressOutput(columnProperties),
       virtual = getVirtual(columnProperties),
+      format = getMaybeFormatForColumn(formatNode),
       textDirection = getTextDirection(inheritedPropertiesCopy),
       annotations = annotations,
-      warnings = warnings
+      warnings = warnings,
+      errors = Array()
     )
   }
 
+  private def getMaybeFormatForColumn(formatNode: JsonNode) = {
+    if (formatNode.isMissingNode || formatNode.isNull) {
+      None
+    } else {
+      formatNode match {
+        case s: TextNode => Some(Format(Some(s.asText()), None, None, None))
```

Comment:
Yeah, I think the format string should go into the `format` property instead of this new special location.

Reply:
Okay, reusing pattern property now.